commands/management/add_mysql.go

```diff
@@ -125,7 +126,8 @@ func (cmd *addMySQLCommand) GetAddress() string {
 }
 
 func (cmd *addMySQLCommand) GetDefaultAddress() string {
-	return "127.0.0.1:3306"
```

Comment:
let's return default value if `cmd.DefaultsFile` is not specified.

Reply:
Done.